server/app/models/TrustedIntermediaryGroupModel.java

```diff
@@ -36,13 +36,22 @@ public ImmutableList<AccountModel> getTrustedIntermediaries() {
     return ImmutableList.copyOf(tiAccounts);
   }
 
+  public int getTrustedIntermediariesSize() {
+    return tiAccounts.size();
+  }
+
   /** Get all the accounts, sorted by applicant name. */
   public ImmutableList<AccountModel> getManagedAccounts() {
     return managedAccounts.stream()
         .sorted(Comparator.comparing(AccountModel::getApplicantName))
         .collect(ImmutableList.toImmutableList());
   }
 
+  /** Gets the size of Managed Accounts */
+  public int getManagedAccountsSize() {
```

Comment:
(suggestion) Call this `getManagedAccountsCount`. Size is more a list thing, but count makes more sense if you're thinking about it from the table on the page.

Reply:
done